src/Umbraco.Web/Trees/ContentTreeControllerBase.cs

```diff
@@ -203,7 +203,9 @@ protected IEnumerable<IUmbracoEntity> GetChildEntities(string id)
                 entityId = entity.Id;
             }
 
-            return Services.EntityService.GetChildren(entityId, UmbracoObjectType).ToArray();
+            var v2 = Services.EntityService.GetChildren(entityId, UmbracoObjectType).ToList();

```

Comment:
Hi, can we rename the variable to something more meaningful?

Reply:
Haha yes, my bad, just a leftover debug artifact.
Thanks for letting me know :D.